lib/umwelt/files.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Umwelt.Files do
+  @moduledoc "Finds elixir files recursive from dir"
+
+  def list_from_root(project),
+    do: files_in("lib/#{project}")
```

Comment:
```suggestion
  def list_from_root(project \\ Mix.Project.config[:app]) do
    Mix.Project.config[:elixirc_paths]
    |> Enum.flat_map(&files_in(Path.join(&1, to_string(project)))
  end
```

1. What if I keep sources in `my_lib`?
2. Project info should be taken from `Mix.Project`
3. `Path.join/2` to avoid pissing off our windows friends.

Reply:
For example, this suggestion broke ability of parser to parse itself. Before commit it I've realised the parsing of default argument, access and function call.

Reply:
> this suggestion broke ability of parser to parse itself

Eh. In what way exactly?

Reply:
Mix task [test](https://github.com/sovetnik/umwelt/pull/1/files#diff-479cb9f082caed0efbb11d242eb259fbde9235c81124f6cb8899bb6a8b98ea70) that ensures parser can parse it lib

Reply:
Parsing of default argument and function call haven't  been implemented in that moment, but now it is [realised](https://github.com/sovetnik/umwelt/blob/e3e3232903bc9058c2040f3dc9a1c1ad5a30111b/test/umwelt/parser/def_test.exs#L41)

Reply:
Ah, got it.

FWIW, to avoid escaping backslashes, use `~S"""` sigil.

Reply:
Thanks! I've needed it.